rhel8/profiles/cis.profile

```diff
@@ -757,7 +757,7 @@ selections:
     - sshd_set_loglevel_info
 
     ### 5.2.6 Ensure SSH X11 forward is diabled (Scored)
-
+    - sshd_disable_x11_forwarding
```

Comment:
Just do the change to add the rule to the profile. Please remove the other formatting changes.
I know they can be annoying but keeping changes to the relevant lines helps to avoid conflicts.

Reply:
reverted to original and added the check - per comment

Reply:
The revert commit didn't revert the formatting changes, hehe.

Reply:
Dang it. I pulled it out of my master which is synced upstream to you. I will fix it. Sorry that I am a noob

Reply:
Pulled from upstream and edited the file.  Hopefully this fixes it.

Reply:
> Dang it. I pulled it out of my master which is synced upstream to you. I will fix it. Sorry that I am a noob

Apologies for my delay.
Don't worry, happy that you are contributing.

If you are willing to go one step further, we can make the git history cleaner, :).
The last 4 commits can be squashed together, as they are all attempting to do the same thing.
You can use command: `git rebase -i HEAD~4` and squash the commits.

For reference: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History , section `Squashing Commits` has more details...

Reply:
Unresolved for visibility purposes.

Reply:
I think I accomplished what you asked.  Once again, thanks for bearing with and educating me.